cpp/include/cudf/strings/char_types/char_types.hpp

```diff
@@ -82,12 +90,12 @@ string_character_types& operator|=(string_character_types& lhs, string_character
  *
  * Any null row results in a null entry for that row in the output column.
  *
- * @param strings Strings instance for this operation.
- * @param types The character types to check in each string.
- * @param verify_types Only verify against these character types.
+ * @param strings Strings instance for this operation
+ * @param types The character types to check in each string
+ * @param verify_types Only verify against these character types
```

Comment:
```suggestion
 * @param verify_types Only verify against these character types.
```
Doxygen will render this together so there needs to be a break after `types.` 
![image](https://user-images.githubusercontent.com/45795991/170260878-96b7b714-f50b-4483-be10-10dc87f04604.png)